packages/api-plugin-fulfillment/src/util/getCartById.js

```diff
@@ -0,0 +1,34 @@
+import hashToken from "@reactioncommerce/api-utils/hashToken.js";
+import ReactionError from "@reactioncommerce/reaction-error";
```

Comment:
Confused why this is here and not in api-carts?

Reply:
It was copied over from the old shipments plugin. As discussed, we have the same function inside api-carts also. So I shall make a new query for the one in api-cart and remove this one

Reply:
Since the changes will be spread over multiple PRs if implemented now, I shall wait for the currents PRs to be merged and create a new PR for this change.

Reply:
Implemented this change.
added the api-cart copy to queries and deleted api-fulfillment/src/util version